hindi/swami-vivekananda/swami-vivekananda brief introduction.md

```diff
@@ -0,0 +1,4 @@
+﻿स्वामी विवेकानन्द (जन्म: १२ जनवरी,१८६३ - मृत्यु: ४ जुलाई,१९०२) वेदान्त के विख्यात और प्रभावशाली आध्यात्मिक गुरु थे।
+उनका वास्तविक नाम नरेन्द्र नाथ दत्त था। उन्होंने अमेरिका स्थित शिकागो में सन् १८९३ में आयोजित विश्व धर्म महासभा में भारत की ओर से सनातन धर्म का
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.